src/utils/pageContext.ts

```diff
@@ -0,0 +1,11 @@
+import { Session } from 'next-auth';
+import React from 'react';
+
+export interface PageContext {
+    theme: 'light' | 'dark';
+    themeId: number;
+    ssrTime: number;
+    user?: Session['user'];
+}
+
+export const pageContext = React.createContext<PageContext>({ theme: 'dark', themeId: 0, ssrTime: 0 });
```

Comment:
This context is not rendered anywhere, it will not work

Better to do this in separate issue